src/predefined-variables.ts

```diff
@@ -41,6 +41,7 @@ export function init ({gitData, argv, envMatchedVariables}: PredefinedVariablesO
       ?? `${CI_SERVER_PROTOCOL}://${CI_SERVER_FQDN}`;
     const CI_PROJECT_ROOT_NAMESPACE = gitData.remote.group.split("/")[0];
     const CI_PROJECT_NAMESPACE = gitData.remote.group;
+    const CI_DEPENDENCY_PROXY_SERVER = `${CI_SERVER_HOST}:${CI_SERVER_PORT}`; // NOTE: unlike CI_SERVER_FQDN, CI_DEPENDENCY_PROXY_SERVER also has ports
```

Comment:
This comment isn't correct. `CI_SERVER_FQDN` does include the port, so we can use that here.
```suggestion
    const CI_DEPENDENCY_PROXY_SERVER = `${CI_SERVER_FQDN}`;
```

Reply:
i actually meant CI_DEPENDENCY_PROXY_SERVER **always** has ports, 

anyways updated the code, hopefully it's more self explanatory